web-common/src/features/sources/workspace/SourceWorkspaceHeader.svelte

```diff
@@ -115,13 +115,18 @@
   }
 
   const onSaveAndRefreshClick = async (tableName: string) => {
+    // no-op if the clientYAML is still null
+    if ($sourceStore.clientYAML === null) return;
```

Comment:
Similar to above. If `clientYAML` is null, can we instead disable the Button so this code path never gets hit?